postgres/coredb-pg-slim/postgresql.conf

```diff
@@ -840,3 +840,5 @@ pg_partman_bgw.interval = 15
 pg_partman_bgw.role = 'postgres'
 restore_command = 'wal-g wal-fetch "%f" "%p"'
 recovery_target_timeline = 'latest'
+
+include_if_exists 'tembo.postgresql.conf'
```

Comment:
per our conversation, change this to be `include_if_exists '/tembo/config/tembo.postgresql.conf`